pkg/trace/api/normalizer.go

```diff
@@ -123,6 +118,15 @@ func normalize(ts *info.TagStats, s *pb.Span) error {
 		log.Debugf("Fixing malformed trace. Duration is too large and causes overflow (reason:invalid_duration), setting span.duration=0: %s", s)
 		s.Duration = 0
 	}
+	if s.Start < Year2000NanosecTS {
```

Comment:
Can we change line 116 to use math.MaxInt64 for the overflow check?

Reply:
Yeah I guess that's a bit clear than actually causing an overflow to check it. Updated